nvtabular/loader/torch.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         device=None,
         global_size=None,
         global_rank=None,
+        drop_last=False,
```

Comment:
I think this covers it for Torch, but just double-checking: Do we need to any additional changes here to parallel the TF changes (e.g. with stopping the dataloader)?

Reply:
No because the pytorch dataloader actually runs using the iterator logic as opposed to get_item and len...